src/flakehub_auth_fake.rs

```diff
@@ -0,0 +1,45 @@
+use color_eyre::eyre::{Context, Result};
+
+use crate::github::graphql::GithubGraphqlDataResult;
+
+pub async fn get_fake_bearer_token(
+    jwt_issuer_uri: &url::Url,
```

Comment:
I think this needs to take the issuer as `&str` -- `url::Url` adds a `/` at the end, but fakeidp doesn't like that because it treats URLs as strings